src/types/video.ts

```diff
@@ -61,4 +61,42 @@ export type UnprocessedVideoInfo = {
    *  The height of the video
    */
   vHeight: number
+  /**
+   *  Fist chunk of the video
+   */
+  thumbnail: string | undefined
+}
+
+/* eslint-disable jsdoc/require-jsdoc  */
+export type VideoLibraryFile = {
+  fileName: string
+  size?: number
+  url: string
+  dateStart?: Date
+  dateLastRecordingUpdate?: Date
+  dateFinish?: Date
+  vWidth?: number
+  vHeight?: number
+  hash?: string
+  thumbnail?: string
+  isProcessed: boolean
+}
+
+export type VideoLibraryLogFile = {
+  fileName: string
+  size?: number
+  url?: string
+  dateStart?: Date
+  dateLastRecordingUpdate?: Date
+  dateFinish?: Date
+  vWidth?: number
+  vHeight?: number
+  hash?: string
+  thumbnail?: string
+}
```

Comment:
I would suggest extending those two from the `UnprocessedVideoInfo` type. Right now we have the code triplicated on those types.

Reply:
Just to be sure, you meant separate commit or separate PR? If you prefer, I think it's fine to leave this here for now.